public/components/datasources/components/manage/accelerations/create_accelerations_flyout/selectors/source_selector.tsx

```diff
@@ -74,7 +74,11 @@ export const AccelerationDataSourceSelector = ({
   const loadDataSource = () => {
     setLoadingComboBoxes({ ...loadingComboBoxes, dataSource: true });
     http
-      .get(DATACONNECTIONS_BASE + `/dataSourceMDSId=${dataSourceMDSId}`)
+      .get(
+        dataSourceMDSId
+          ? `${DATACONNECTIONS_BASE}/dataSourceMDSId=${dataSourceMDSId}`
```

Comment:
Can we do url concatenation outside of get() to make it clearer? And similar here: https://github.com/opensearch-project/dashboards-observability/pull/1830/files#diff-26331e46c0a085042595fee8ac2c10aac6be43368989e91c38b9e8572d71c7bfR41